code/modules/events/false_alarm.dm

```diff
@@ -11,11 +11,18 @@
 /datum/round_event/falsealarm/announce()
 	var/list/events_list = list()
 	for(var/datum/round_event_control/E in SSevent.control)
-		if(!E.wizardevent && !E.holidayID) //No holiday cheer allowed during non-holidays. Not even fake holiday cheer.
-			events_list += E //No holiday cheer allowed during non-holidays. Not even fake holiday cheer.
+		if(E.holidayID || E.wizardevent)
+			continue
+		var/datum/round_event/event = E.typepath
+		if (initial(E.typepath) < 0)
+			continue

```

Comment:
I think you made a mistake, you're making an "event" var and do not use it, and E.typepath is a type so why compare it to a number?

This should probably be
```
var/datum/round_event/event = E.typepath
if(initial(event.announceWhen) <= 0)
	continue
```